### ISSUE TITLE
Changing the orientation of a player's collision box

### ISSUE THREAD

**Issue by YourStalker** [Facepunch/garrysmod-requests#650]

**Comment by Kefta** [comment]
Couldn't you just set the bounds to "rotate it"

**Comment by YourStalker** [comment]
Yeah but if you are looking at one of the corner's of the map then what? You would line up diagonally with your collision box and there would be two corners on the left and right sticking out. It also doesn't help that SetHull isn't networked.

**Comment by neico** [comment]
SetSolid( SOLID_OBB_YAW ) is a fun thing to do

That's about the only method I know how to make it "rotate" (it rotates left and right relative to the player model)
